src/metrics/function_error.py

```diff
@@ -37,6 +37,28 @@ def evaluate(self, models: Iterable[ContextModel], num_batches: int = 1) -> Iter
 
         return errs
 
+class RegressionScore(Benchmark):
```

Comment:
Since this is not a function error, can you move this to a separate file? It'll make adding new function errors a little easier in the coming days